### DISCUSSION TITLE
Vercel website

### DISCUSSION THREAD

**Discussion by dev-030** [vercel/community#2442] [Bug Report]
### Summary

When i try to go vercel dashboard , sometimes i face errors and the website just doesnt load and sometimes i doesnt give any errors , everything just works fine as expected . It also happens to the deployments i have , some times i can visit them or sometimes i cannot , it just gives errors , its not an network issue , i have tried other providers too, netlify , render etc , it doesnt happens with them , only with vercel . But when i connect a vpn , every thing just works fine and smooth .

**Comment by SwimmingHwang** [comment]
Me too

**Comment by Pieparker** [comment]
Would y'all be able to share specific error messages or screenshots (removing any possible sensitive data) with us? If you can share your ISP and country where your requests are being made, this could help too.
